stacks_queues/queue.py

```diff
@@ -1,61 +1,93 @@
-
 INITIAL_QUEUE_SIZE = 20
 
+
 class QueueFullException(Exception):
     pass
 
+
 class QueueEmptyException(Exception):
     pass
 
-class Queue:
 
+class Queue:
     def __init__(self):
         self.store = [None] * INITIAL_QUEUE_SIZE
         self.buffer_size = INITIAL_QUEUE_SIZE
         self.front = -1
         self.rear = -1
         self.size = 0
-      
 
     def enqueue(self, element):
-        """ Adds an element to the Queue
-            Raises a QueueFullException if all elements
-            In the store are occupied
-            returns None
+        """Adds an element to the Queue
+        Raises a QueueFullException if all elements
+        In the store are occupied
+        returns None
         """
-        pass
+        if self.size == INITIAL_QUEUE_SIZE:
```

Comment:
I suggest always comparing `size` to `buffer_size` here instead of `INITIAL_QUEUE_SIZE`. Imagine you adapt your code such that the buffer size has the ability to change. You would have to manually update your code to refer to the new buffer size instead of the initial size.
```suggestion
        if self.size == self.buffer_size:
```